### PR TITLE
Update around LIFF API

### PR DESCRIPTION
**Pull request by sugyan**
- Add `liff_test.go`
- Fix `liff.go`
- Remove unnecessary comments

### CODE REVIEW

File: linebot/liff.go [hunk 1]
```diff
@@ -195,7 +204,7 @@ func (call *DeleteLIFFCall) WithContext(ctx context.Context) *DeleteLIFFCall {
 
 // Do method
 func (call *DeleteLIFFCall) Do() (*BasicResponse, error) {
-	endpoint := fmt.Sprintf(APIEndpointDeleteLIFFAPP, call.LIFFID)
+	endpoint := fmt.Sprintf(APIEndpointDeleteLIFFAPP, call.liffID)
```
Comment (mokejp) [inline linebot/liff.go]: I also want to rename `APIEndpoint*LIFFAPP` constants to `APIEndpoint*LIFFApps`.

Comment (sugyan) [reply linebot/liff.go]: Ah, I agree. I'll change them, thanks!

**Comment by suzuki-shunsuke** [comment]
Note that this PR may include breaking changes.

**Comment by sugyan** [comment]
I'll write about breaking change in https://github.com/line/line-bot-sdk-go/tags. 🙇

**Pull request merged**